cmd/controller/cmd/root.go

```diff
@@ -223,6 +223,13 @@ func executeRootCmd(cfg *config2.Config) {
 		logger.Fatalf(ctx, "Failed to start Controller, nil controller received.")
 	}
 
+	go func() {
```

Comment:
I merged that change to master... when you rebase you will have it.

Reply:
sure will do